tests/pytests/test_aux_save2.py

```diff
@@ -138,12 +138,9 @@ def testLoadRdbWithEmptySpellcheckDict(env):
     dbFileName = env.cmd('config', 'get', 'dbfilename')[1]
```

Comment:
Can you also fix the test so we don't override the `env` right away? (remove the `env = Env()` line or the test `env` parameter